### DISCUSSION TITLE
Logo license?

### DISCUSSION THREAD

**Discussion by m1212e** [prisma/prisma#17214] [Q&A] [answered]
Hi,
under what license is the prisma logo/trademark? I'd like to mention that I worked with prisma in my resume, including the logo? Is that allowed?

**Comment by nurul3101** [comment, marked as answer]
Hello @m1212e 👋 
I believe there shouldn't be an issue with using prisma logo in your resume.

Prisma is licensed under [Apache License 2.0](https://github.com/prisma/prisma/blob/main/LICENSE).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
